core/flyout.js

```diff
@@ -259,47 +328,147 @@ Blockly.Flyout.prototype.position = function() {
     // Hidden components will return null.
     return;
   }
-  var edgeWidth = this.width_ - this.CORNER_RADIUS;
-  if (this.RTL) {
+  var edgeWidth = this.horizontalLayout_ ? metrics.viewWidth : this.width_;
```

Comment:
is the difference between viewWidth and this.width the scrollbar padding?  And is scrollbar padding applied on the sides of the flyout without the scrollbars?  Sorry for being so confused about the meaning of  SCROLLBAR_PADDING.